pystac_monty/sources/usgs.py

```diff
@@ -403,7 +404,13 @@ def make_hazard_event_item(self) -> Item:
         hazard_item.id = f"{STAC_HAZARD_ID_PREFIX}{hazard_item.id.replace(STAC_EVENT_ID_PREFIX, '')}-shakemap"
 
         # extent the hazard zone with the shakemap extent
-        shakemap = self.data.get_data()["properties"].get("products", {}).get("shakemap", [])[0]
+        shakemap = self.data.get_data()["properties"].get("products", {}).get("shakemap", [])
+
+        if not shakemap:
+            shakemap = {}
+        else:
+            shakemap = shakemap[0]
```

Comment:
This can still be None. So we should be careful if that is what we want.

Reply:
Done